### PR TITLE
Add Dai Specs

### PR DESCRIPTION
**Pull request by naszam**

### CODE REVIEW

**Comment by naszam** [comment]
## Run Dai Specs

* certora-cli >= 2.7.1
	* `pip3 install certora-cli==2.7.1` 
* Switch to solc 0.6.11
```
pip3 install solc-select && solc-select install 0.6.11
```

* **Dai**
```
yarn certora
```
[Report](https://prover.certora.com/output/42696/8ca966aa95e0b1087bf2/?anonymousKey=f1958c0f44dec847b02fdfb8a86a35f7f24bff22)

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "format": "yarn workspace bridge format",
     "test": "yarn workspace bridge test",
     "test-e2e": "yarn workspace bridge test-e2e",
-    "test:fix": "yarn workspace bridge test:fix"
+    "test:fix": "yarn workspace bridge test:fix",
+    "certora": "certoraRun --solc ~/.solc-select/artifacts/solc-0.6.11 contracts/l2/dai.sol:Dai certora/HashHelper.sol --verify Dai:certora/dai.spec --rule_sanity --solc_args \"['--optimize','--optimize-runs','200']\""
```
Comment (hexonaut) [inline package.json:19]: This should be changed to `"certora": "yarn workspace bridge certora"`

**Review by hexonaut — CHANGES_REQUESTED** [review]
Small fix in root `package.json`, but other than that tests are passing. Good to merge after the fix.

File: package.json (hunk 1 above)
Comment (naszam) [reply package.json]: Done in 2a5223d.

**Review by hexonaut — APPROVED** [review]
Good to merge.

**Pull request merged**